addons/im_livechat/models/discuss_channel.py

```diff
@@ -96,6 +96,37 @@ def _gc_empty_livechat_sessions(self):
     def execute_command_history(self, **kwargs):
         self._bus_send("im_livechat.history_command", {"id": self.id})
 
+    def execute_command_bot(self, bot_id, **kwargs):
+        if not self.livechat_active:
+            # To-do: remove this after livechat visitor leave has been adapted to use action_unfollow
+            return
+        chatbot_script = self.env['chatbot.script'].browse(bot_id)
+        if not self or not chatbot_script.exists():
+            self._send_transient_message(self.env.user.partner_id, _("Please select a chatbot"))
+            return
+        if self.livechat_operator_id != chatbot_script.operator_partner_id:
+            operator_partner = self.env['chatbot.script'].search(
+                [('operator_partner_id', '=', self.livechat_operator_id.id)],
+                limit=1
+            ).operator_partner_id
+            if operator_partner:
+                # sudo - discuss.channel: non admin operator can remove other operators
```

Comment:
I find the naming of `operator_partner_id` field confusing, and the sudo comment makes it even more so. Here, `operator_partner_id` refers to another bot that was already started on the channel. I think a comment mentioning that another bot was already activated on the channel would improve understanding of this part.